lib/list.gd

```diff
@@ -2376,36 +2376,22 @@ DeclareGlobalFunction( "ListWithIdenticalEntries" );
 ##
 #F  PlainListCopy( <list> ) . . . . . . . .  make a plain list copy of a list
 ##
+##  <#GAPDoc Label="PlainListCopy">
 ##  <ManSection>
 ##  <Func Name="PlainListCopy" Arg='list'/>
 ##
 ##  <Description>
+##  This operation returns a list equal to its argument, in a plain list
+##  representation.
 ##  This is intended for use in certain rare situations,
-##  such as before objectifying.
+##  such as before objectifying or calling some kernel functions.
 ##  Normally, <C>ConstantAccessTimeList</C> should be enough.
```

Comment:
I know you didn't write this, but I can't find any other reference to `ConstantAccessTimeList` in the whole of GAP... so I'm not sure what this is referring to. Maybe just delete this line?
```suggestion
```